packages/util/src/assets/index.ts

```diff
@@ -375,38 +375,40 @@ export class AssetsModule {
     name: string,
     totalSupply: NumberLike,
     extensibleSupply: boolean,
+    nonDivisible: boolean,
     nft = {
-      isNft: false,
       content: null,
       description: null,
     }
   ) {
     assert(this.root.account, Messages.connectWallet);
-    const supply = nft.isNft ? new FPNumber(totalSupply, 0) : new FPNumber(totalSupply);
+    const supply = nonDivisible ? new FPNumber(totalSupply, 0) : new FPNumber(totalSupply);
     return {
-      args: [symbol, name, supply.toCodecString(), extensibleSupply, nft.isNft, nft.content, nft.description],
+      args: [symbol, name, supply.toCodecString(), extensibleSupply, nonDivisible, nft.content, nft.description],
     };
   }
 
   /**
    * Register asset
    * @param symbol string with asset symbol
    * @param name string with asset name
-   * @param totalSupply
-   * @param extensibleSupply
+   * @param totalSupply total token supply
+   * @param extensibleSupply `true` means that token can be mintable any time by the owner of that token. `false` by default
+   * @param nonDivisible `false` by default
+   * @param nft Nft params object which contains content & description
    */
   public async register(
     symbol: string,
     name: string,
     totalSupply: NumberLike,
     extensibleSupply = false,
+    nonDivisible = false,
```

Comment:
I thought most of the time it would be non-divisible

Reply:
divisible = 18 decimals